cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/BuildCommand.java

```diff
@@ -144,6 +144,9 @@ public BuildCommand(Path userDir, PrintStream outStream, PrintStream errStream,
     @CommandLine.Option(names = "--dump-llvm-ir", hidden = true)
     private boolean dumpLLVMIR;
 
+    @CommandLine.Option(names = "--no-optimize-llvm", hidden = true)
+    private boolean noOptimizeLlvm;
+
```

Comment:
shall we change variable name as below?
noOptimizeLlvm -> noOptimizeLLVM

Reply:
https://stackoverflow.com/questions/15526107/acronyms-in-camelcase

Reply:
Refactoring in a seperate PR